Registry/Registry.EM_COMMON

```diff
@@ -339,13 +339,13 @@ i1       real   tke_tend       ikj     dyn_em      1         -
 
 # Pressure and Density
 state    real   p              ikj     dyn_em      1         -      irh       "p"           "perturbation pressure"         "Pa"
-state    real   al             ikj     dyn_em      1         -      r         "al"          "inverse perturbation density"  "m3 kg-1"
+state    real   al             ikj     dyn_em      1         -      ir        "al"          "inverse perturbation density"  "m3 kg-1"
```

Comment:
@smileMchen @weiwangncar @dudhia 
Ming,
We now are going to assume that al and alb are input from the real program. If someone is using older data, say v4.0 instead of v4.1, does this mean that some of the diagnostics at the initial time will be incorrect? 

Wei and Jimy,
Do we go to the trouble of testing IC data versions for the initial time of a diagnostic? This test would need to be in input_wrf.F not check_a_mundo.F

Reply:
@davegill 
(1) al and alb won't affect any diagnostics in  previous versions of WRF. 
(2) They only affect diagnostics of traditional fields (added by this PR) at the initial time. This is why they are included in wrfinput (as proposed by this PR).